src/orion/client/experiment.py

```diff
@@ -581,7 +581,9 @@ def observe(self, trial, results):
 
         """
         self._check_if_executable()
-
+        if not isinstance(results, list):
```

Comment:
That'll be convenient! Needs testing however.

Reply:
Where/How? Can I leave that for another PR?

Reply:
Tests would be here: https://github.com/Epistimio/orion/blob/develop/tests/unittests/client/test_experiment_client.py#L744. Yes that could be for another PR, but the if-clause should also be added in that other PR otherwise we would introduce the new feature without testing it.